tests/testConvolve.py

```diff
@@ -39,7 +39,7 @@
 import numpy
 
 import lsst.utils
-import lsst.utils.tests as utilsTests
+import lsst.utils.tests
 import lsst.pex.logging as pexLog
 import lsst.afw.geom as afwGeom
 import lsst.afw.image as afwImage
```

Comment:
I suspect tests/kernel.py was renamed to tests/testKernel.py, which would have broken this. It would be safer to move the shared code to afw test utils, but that may be out of scope for this ticket.